packages/client/src/components/ModuleTree/Module/Icons/circle.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export const Circle: React.FunctionComponent<{className?: string}> = ({ className, ...props }) => (
```

Comment:
Are we ever going to need these icons in ADMIN @tristanMatthias? If so we should move this to UI package. Same applies for hexagon and octagon.

Reply:
Yeah possibly. Let's move to `ui` :)